### DISCUSSION TITLE
merge conflicts in main branch

### DISCUSSION THREAD

**Discussion by akshaysharmajs** [Jshel/CEN5035Project#62] [General]
`Error: src/app/create-account/create-account.component.ts:13:1 - error TS1185: Merge conflict marker encountered.

13 <<<<<<< HEAD
   ~~~~~~~


Error: src/app/create-account/create-account.component.ts:15:1 - error TS1185: Merge conflict marker encountered.

15 =======
   ~~~~~~~

Error: src/app/create-account/create-account.component.ts:17:1 - error TS1185: Merge conflict marker encountered.

17 >>>>>>> 2961a4c8b9a0388286cb481a5aa2dfc18f7526da
   ~~~~~~~


Error: src/app/create-account/create-account.component.ts:30:1 - error TS1185: Merge conflict marker encountered.

30 <<<<<<< HEAD
   ~~~~~~~

Error: src/app/create-account/create-account.component.ts:37:1 - error TS1185: Merge conflict marker encountered.

37 =======
   ~~~~~~~


Error: src/app/create-account/create-account.component.ts:44:1 - error TS1185: Merge conflict marker encountered.

44 >>>>>>> 2961a4c8b9a0388286cb481a5aa2dfc18f7526da
   ~~~~~~~

Error: src/app/login-form/login-form.component.ts:13:1 - error TS1185: Merge conflict marker encountered.

13 <<<<<<< HEAD
   ~~~~~~~


Error: src/app/login-form/login-form.component.ts:16:1 - error TS1185: Merge conflict marker encountered.

16 =======
   ~~~~~~~

Error: src/app/login-form/login-form.component.ts:21:1 - error TS1185: Merge conflict marker encountered.

21 >>>>>>> 2961a4c8b9a0388286cb481a5aa2dfc18f7526da
   ~~~~~~~


Error: src/app/login-form/login-form.component.ts:33:1 - error TS1185: Merge conflict marker encountered.

33 <<<<<<< HEAD
   ~~~~~~~

Error: src/app/login-form/login-form.component.ts:40:1 - error TS1185: Merge conflict marker encountered.

40 =======
   ~~~~~~~


Error: src/app/login-form/login-form.component.ts:47:1 - error TS1185: Merge conflict marker encountered.

47 >>>>>>> 2961a4c8b9a0388286cb481a5aa2dfc18f7526da
   ~~~~~~~

** Angular Live Development Server is listening on localhost:4200, open your browser on http://localhost:4200/ **

✖ Failed to compile.
`

**Comment by Jshel** [comment]
I think the 58 branch has the most recent pushes to the frontend.
